.github/workflows/tests.yaml

```diff
@@ -0,0 +1,30 @@
+name: Tests
+
+on:
+  push:
+  pull_request:
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        # Run tests across all platforms, to ensure developers
+        # can use any environment to run basic tests.
+        platform: [ubuntu-latest, windows-latest, macos-latest]
+
+    runs-on: ${{ matrix.platform }}
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+
+    - name: Install Go
+      uses: actions/setup-go@v5
+      with:
+        go-version: 'stable'
```

Comment:
Do we know what 'stable' really translates to on any given run? Should this value match the one found in go.mod?

Reply:
That's a good point. I'm going to use the version from go.mod instead.